tensorrt_llm/bench/benchmark/throughput.py

```diff
@@ -467,10 +466,6 @@ def ignore_trt_only_args(kwargs: dict):
         report_utility.report_statistics()
     except KeyboardInterrupt:
         logger.info("Keyboard interrupt, exiting benchmark...")
-        sys.exit(130)
```

Comment:
Is it expected to remove the `exit` for `KeyboardInterrupt`?

Reply:
That or try and shutdown the backend task properly.

Reply:
Got it. Free to fix this in a subsequent PR if CI passed.